tests/test_lispify.py

```diff
@@ -19,81 +21,101 @@
 class TestLispify(unittest.TestCase):
 
     def test_string(self):
-        self.assertEqual(lispify("foo"), '"foo"')
+        self.assertEqual(str(lispify("foo")),
+                         '"foo"')
 
     def test_string_escaped(self):
-        self.assertEqual(lispify('foo \\ "bar"'), '"foo \\ \\"bar\\""')
+        self.assertEqual(str(lispify('foo \\ "bar"')),
+                         '"foo \\ \\"bar\\""')
+
+    @unittest.skipIf(version_info.major == 2, 'python 3 string behavior')
```

Comment:
Can you make this one a `skipUnless` just to be consistent with the below one? Alternatively, make both `skipIf` -- your choice.